### PR TITLE
Updated SocialActivityAttachment and SocialActivityModel classes to implement ImportableItem class

### PR DESCRIPTION
**Pull request by saratchebrolu**
Updated SocialActivityAttachment and SocialActivityModel classes to implement ImportableItem class

### CODE REVIEW

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityAttachment.java [hunk 1]
```diff
@@ -70,4 +74,8 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(getType(), getUrl(), getName(), getContent());
   }
+
+  @Nullable
```
Comment (alexeyqu-fb) [inline portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityAttachment.java:78]: I suppose `getIdempotentId` should be `@Nonnull` here?

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityModel.java [hunk 2]
```diff
@@ -114,4 +116,14 @@ public String getContent() {
   public String getUrl() {
     return url;
   }
+
+  @Nullable
```
Comment (alexeyqu-fb) [inline portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityModel.java:120]: ditto

**Review by alexeyqu-fb — CHANGES_REQUESTED** [review]
Not sure about the `Nullable` usage for `getIdempotentId` method, is it correct?

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityAttachment.java (hunk 1 above)
Comment (saratchebrolu) [reply portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityAttachment.java]: Yes, updated it

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityModel.java [hunk 3]
```diff
@@ -19,12 +19,15 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.ImmutableList;
+import org.datatransferproject.types.common.ImportableItem;
 
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
```
Comment (alexeyqu-fb) [inline portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityModel.java:25]: don't need to import `Nullable` now

**Review by alexeyqu-fb — CHANGES_REQUESTED** [review]
tiny nit left

**Review by alexeyqu-fb — APPROVED** [review]
LGTM

**Pull request merged**